codegen/src/main/kotlin/org/web3j/openapi/codegen/config/ContractConfiguration.kt

```diff
@@ -16,6 +16,10 @@ import java.io.File
 
 data class ContractConfiguration(
     val abiFile: File,
-    val binFile: File,
+    val binFile: File?,
     val contractDetails: ContractDetails
-)
+) {
+    init {
+        if (binFile == null) println("${abiFile.nameWithoutExtension} contract address will be expected to be defined later !")
```

Comment:
Why is that for? If there are requirements that should be checked in the Pico CLI command.

@iikirilov is in the process of putting the commands back to the console module.